commands/volumes/upload.go

```diff
@@ -56,28 +58,64 @@ func handleUploadVolumeCmd(c client.Client) error {
 		params["Org"] = c.Org
 	}
 
+	var file *os.File
+	var err error
+
+	filename := viper.GetString("path")
+	fi, err := os.Stat(filename)
```

Comment:
You can use the loop to set the string and just call 
`	fi, err := os.Stat(filename)
	if err != nil {
		return err
	}
` 
once, then defer file.Close() afterwards

Reply:
Not sure what loop you're referring to. I refactored the logic in the `switch` statement later.